mlir/include/mlir/Interfaces/ValueBoundsOpInterface.h

```diff
@@ -313,7 +313,8 @@ class ValueBoundsConstraintSet
   /// An index-typed value or the dimension of a shaped-type value.
   using ValueDim = std::pair<Value, int64_t>;
 
-  ValueBoundsConstraintSet(MLIRContext *ctx, StopConditionFn stopCondition);
+  ValueBoundsConstraintSet(MLIRContext *ctx, StopConditionFn stopCondition,
+                           bool addConservativeSemiAffineBounds = false);
```

Comment:
This seems fine to me. Curious what @matthias-springer thinks of adding this to the API.

Reply:
AFAIK this is a semi-private constructor (as most `ValueBoundsConstraintSet` methods are static and construct the actual constraints set internally). So the main use for this was just to allow the `ScalableValueBoundsConstraintSet` to enable this feature.